docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+    browser_fpm:
+        image: ${DOCKER_NAMESPACE}/browser:${REVISION_BROWSER}
+    web:
```

Comment:
Planning on 1 Nginx?

Reply:
1 public nginx due to a single ip and port combination being exposed to the end user, but possible more internally that are not exposed but act as a gateway or as the reverse proxy for a service.